CONTRIBUTING.md

```diff
@@ -1,256 +1,339 @@
 
-__NOTE__: Automatic fix version doesn't work if package is using the _revision_ part of the version (4th number). In that case _explicit version_ must be used: `[AU package:version]`.
-## 1.4 UI Automation
+**IMPORTANT: If there is no icon available, the comment `<!-- IconUrl: Skip check -->` is required to be specified in the metadata file**
+
+#### Chocolatey compatibility
+
+All packages are expected to be compatible with the version of Chocolatey CLI that was available 12 months earlier than the last change of the package.
+With that, it means that if you make a change to the nuspec, or one of the package scripts on January 12th, 2023 it should be compatible with Chocolatey CLI 0.11.3 and later released versions.
+If that is not possible, and the package needs a version of Chocolatey CLI that was released later, it will need to include a dependency for that Chocolatey CLI version.
+
+### Chocolatey Scripts
+
+All PowerShell scripts are expected to be compatible with PowerShell v2 and later (not PowerShel Core). If this is not possible, a dependency on the package `PowerShell` is required with its minimum version set to the earliest version that the scripts in the package is expected to work with.
+
+**TODO: This section needs to be updated, and is a copy-paste from earlier guidelines**
+
+#### UI Automation
 
 Some installers do not provide silent arguments and can be difficult to automate.
 
 If the package needs to perform tasks that cannot be done with command line switches, e.g. clicking away a prompt during installation that cannot be suppressed like for example in the [dropbox](https://chocolatey.org/packages/dropbox) package, you can use [AutoHotkey](https://chocolatey.org/packages/autohotkey.portable). Make [autohotkey.portable](https://chocolatey.org/packages/autohotkey.portable) a dependency of the package. Prefer AutoHotkey over [AutoIt](https://chocolatey.org/packages/autoit.commandline), because AutoHotkey is more lightweight, FOS and more actively developed than AutoIt.
 
-### 1.4.1 Work on all locales
+##### Work on all locales
 
 Script must work on every locale available for Windows, so be careful when using strings of text of window in the script that could be different in another language.
 
-### 1.4.2 Avoid brittle scripts
+##### Avoid brittle scripts
 
 Do not create brittle scripts that work only when user doesn't interfer. All script elements should be as precise as possible - for instance, instead of using [Send](https://autohotkey.com/docs/commands/Send.htm) function which will work correctly only if the desired window is active, use [ControlSend](https://autohotkey.com/docs/commands/ControlSend.htm) which doesn't require window activation or use [BlockInput](https://autohotkey.com/docs/commands/BlockInput.htm) for short period of time.
```

Comment:
```suggestion
Do not create brittle scripts that work only when user doesn't interfere. All script elements should be as precise as possible - for instance, instead of using [Send](https://autohotkey.com/docs/commands/Send.htm) function which will work correctly only if the desired window is active, use [ControlSend](https://autohotkey.com/docs/commands/ControlSend.htm) which doesn't require window activation or use [BlockInput](https://autohotkey.com/docs/commands/BlockInput.htm) for short periods of time.
```